install.sh

```diff
@@ -1,32 +1,21 @@
 #!/bin/sh -e
 
-WORKSPACE="${GITHUB_WORKSPACE:=/usr/local}"
 WORKSPACE="/usr/local"
-VERSION=${version:=v0.1.0-alpha.2}
```

Comment:
This is for using the script outside of a gh action and should be kept with the appropriate version